src/main/java/io/opensergo/OpenSergoConfigKindRegistry.java

```diff
@@ -42,6 +43,7 @@ public final class OpenSergoConfigKindRegistry {
         registerConfigKind(ConfigKind.THROTTLING_STRATEGY, ThrottlingStrategy.class);
         registerConfigKind(ConfigKind.CONCURRENCY_LIMIT_STRATEGY, ConcurrencyLimitStrategy.class);
         registerConfigKind(ConfigKind.CIRCUIT_BREAKER_STRATEGY, CircuitBreakerStrategy.class);
+        registerConfigKind(ConfigKind.TRAFFIC_ROUTER_STRATEGY, RouteConfiguration.class);
```

Comment:
We may need to add a comment for this (we've reused the route proto from xDS proto).

Reply:
@jnan806 Any other suggestions?